cmd/chainbridge/main.go

```diff
@@ -41,25 +44,40 @@ var devFlags = []cli.Flag{
 }
 
 var accountCommand = cli.Command{
-	Action:   handleAccountsCmd,
 	Name:     "accounts",
 	Usage:    "manage bridge keystore",
-	Flags:    accountFlags,
+	Flags:    cliFlags,
```

Comment:
I don't think accounts has any flags at this point. The `cliFlags` should be applied to the root app as they are global to all commands.